IfcCommunicationsAppliance Rail Import BIMQ 2021-07-21T07:21:13.7228368Z (#177)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CommunicationsApplianceTypeAntenna/Documentation.md

> Properties in the property set are applicable to the antenna type of communications appliance.

Reviewer: ```suggestion
Properties common to a antenna. This property set is applied to a type or occurrence of _IfcCommunicationsAppliance_ with predefined type of ANTENNA.
```

> Properties in the property set are applicable for the antenna type of communications appliance.

Reviewer 2: ```suggestion
Properties in the property set are applicable to the antenna type of communications appliance.
```

> Properties common to a antenna. This property set is applied to a type or occurrence of _IfcCommunicationsAppliance_ with predefined type of ANTENNA.

Reviewer 2: ```suggestion
Properties common to an antenna. This property set is applied to a type or occurrence of _IfcCommunicationsAppliance_ with predefined type of ANTENNA.
```

> Properties common to an antenna. This property set is applied to a type or occurrence of _IfcCommunicationsAppliance_ with predefined type of ANTENNA.

Reviewer 2: ```suggestion
Properties common to an antenna. This property set is applied to a type or occurrence of _IfcCommunicationsAppliance_ with the predefined type ANTENNA.
```
